source/slang/slang-ir-transform-params-to-constref.cpp

```diff
@@ -44,58 +41,121 @@ struct TransformParamsToConstRefContext
         return true;
     }
 
-    void rewriteParamUseSitesToSupportConstRefUsage(HashSet<IRParam*>& updatedParams)
+    void rewriteValueUsesToAddrUses(IRInst* newAddrInst)
```

Comment:
Note: These are the kinds of function names I greatly appreciate. Yes, they are wordy, but they are also help to convey exactly what is going on, in a way that terse function names seldom can.